package.json

```diff
@@ -2,35 +2,46 @@
   "name": "webpack-retry-chunk-load-plugin",
   "version": "2.2.0",
   "description": "A webpack plugin to retry loading of chunks that failed to load",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
   "directories": {
     "test": "test"
   },
   "peerDependencies": {
     "webpack": ">=5.0.0"
   },
   "devDependencies": {
-    "codecov": "^3.8.1",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.3",
+    "@types/memory-fs": "^0.3.3",
+    "@types/node": "^16.11.10",
+    "codecov": "^3.8.2",
     "express": "^4.17.1",
-    "html-webpack-plugin": "^5.3.1",
-    "husky": "^5.2.0",
-    "jest": "^26.6.3",
+    "gts": "^3.1.0",
+    "html-webpack-plugin": "^5.5.0",
+    "husky": "^7.0.4",
+    "jest": "^27.3.1",
     "memory-fs": "^0.5.0",
-    "nodemon": "^2.0.7",
-    "pretty-quick": "^3.1.0",
-    "webpack": "^5.27.2",
-    "webpack-cli": "^4.5.0",
-    "xo": "^0.38.2"
+    "nodemon": "^2.0.15",
+    "pretty-quick": "^3.1.2",
+    "ts-jest": "^27.0.7",
+    "typescript": "^4.5.2",
+    "webpack": "^5.64.4",
+    "webpack-cli": "^4.9.1"
   },
   "scripts": {
-    "lint": "xo --fix",
+    "lint": "gts lint",
     "test": "jest",
     "test:watch": "npm t -- --watch",
     "test:ci": "npm t -- --runInBand --coverage --coveragePathIgnorePatterns \"test/integration/utils/webpack.js\"",
```

Comment:
I _think_ this is needed: `test/integration/utils/webpack.js` -> test/integration/utils/webpack.ts

Reply:
went with the latter option - works!